examples/core/pom.xml

```diff
@@ -85,6 +85,7 @@
 <!--                            javax.annotation;version=0.0.0,-->
                             <!-- https://github.com/jhy/jsoup/issues/1616 -->
                             javax.annotation;version=!;resolution:=optional,javax.annotation.meta;version=!;resolution:=optional,
+                            com.adobe.cq.wcm.core.components.models;version="[12.0.0,13.0.0)",
```

Comment:
is this required ?

Reply:
we are consuming wcm models in our WCMUse Pojo classes.